### PR TITLE
Set requests and limits for prod env., min replicas 2

### PR DESCRIPTION
**Pull request by ArtisKrumins**

### CODE REVIEW

**Review by Jancis — APPROVED** [review]
I'm fine with these

File: silta/silta-prod.yml [hunk 1]
```diff
@@ -38,6 +43,17 @@ mariadb:
...
       size: 5Gi
+    resources:
+      requests:
+        cpu: 250m
+        memory: 500M
+      limits:
+        cpu: 1000m
+        memory: 750M
+
+# Consider enabling memcached service
```
Comment (tormi) [inline silta/silta-prod.yml:54]: Should we even enable this by default? See https://github.com/wunderio/drupal-project/issues/298.

**Review by tormi — APPROVED** [review]

File: silta/silta-prod.yml (hunk 1 above)
Comment (tormi) [reply silta/silta-prod.yml:54]: cc: @evilfurryone

**Review by agnis-mateuss — APPROVED** [review]

File: silta/silta-prod.yml (hunk 1 above)
Comment (Jancis) [reply silta/silta-prod.yml:54]: that would be a different issue, this one is about requests and limits for existing servers

Comment (tormi) [reply silta/silta-prod.yml:54]: Ok, thanks.

**Pull request merged**